### PR TITLE
[ngspice] Update to rev 33

### PR DESCRIPTION
**Pull request by marekr**
Update ngspice to rev 33

### CODE REVIEW

File: ports/ngspice/CONTROL [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Source: ngspice
-Version: 32
+Version: 33
 Port-Version: 1
```
Comment (NancyLi1013) [inline ports/ngspice/CONTROL:3]:
```suggestion
```

**Review by NancyLi1013 — COMMENTED** [review]
Could you please help check end line of the format for the patch? Is it LF not CRLF?

**Comment by NancyLi1013** [comment]
For the failure on x86-windows, please run `vcpkg.exe x-add-version ngspice` and submit the changes in `versions` folder.
```
Found the following errors:
Error: While reading versions for port ngspice from file: C:\a\1\s\versions\n-\ngspice.json
       Version `33#1` was not found in versions file.
       Run:

           vcpkg x-add-version ngspice

       to add the new port version.

To attempt to resolve all errors at once, run:

    vcpkg x-add-version --all
```

**Comment by marekr** [comment]
> Could you please help check end line of the format for the patch? Is it LF not CRLF?

@NancyLi1013 It is currently LF, is that a problem?

**Review by NancyLi1013 — APPROVED** [review]

**Comment by NancyLi1013** [comment]
LGTM, thanks for your PR @marekr.

**Pull request merged**